pkg/git/repo.go

```diff
@@ -259,6 +260,15 @@ func (r repo) Clean() error {
 	return os.RemoveAll(r.dir)
 }
 
+// CleanPartially deletes data in the given relative path in the repo with git clean.
+func (r repo) CleanPartially(ctx context.Context, relativePath string) error {
```

Comment:
Tbh I think `partially` is not suitable for the function name; how about renaming it `CleanPath`?

Reply:
@khanhtc1202 
Thank you, got it. Fixed in [ab2b33c](https://github.com/pipe-cd/pipecd/pull/5312/commits/ab2b33c326d2581481523bf43da80afd7242bc55)